/**:
  ros__parameters:
    filter_target_label:
      UNKNOWN : true
      CAR : false
      TRUCK : false
      BUS : false
      TRAILER : false
      MOTORCYCLE : false
      BICYCLE : false
      PEDESTRIAN : false

    filter_settings:
      # polygon overlap based filter
      polygon_overlap_filter:
       enabled: true
      # velocity direction based filter
      lanelet_direction_filter:
        enabled: false
        velocity_yaw_threshold: 0.785398 # [rad] (45 deg)
        object_speed_threshold: 3.0 # [m/s]
